### PR TITLE
Match CARD files

### PR DESCRIPTION
**Pull request by Yanis42**
thanks to pikmin2 for most files and mario kart double dash for ``CARDDelete.c``

ah also there was an unused file in configure.py so I removed it (CARDRename.c)

### CODE REVIEW

File: src/dolphin/card/CARDStat.c [hunk 1]
```diff
@@ -0,0 +1,135 @@
+#include "dolphin/card.h"
+
+static void UpdateIconOffsets(CARDDir* entry, CARDStat* state) {
+    u32 offset;
+    bool iconTlut;
+    int i;
+
+    offset = entry->iconAddr;
+    if (offset == 0xffffffff) {
```
Comment (cadmic) [inline src/dolphin/card/CARDStat.c:9]: 0xffffffff -> 0xFFFFFFFF here too?

File: src/dolphin/card/CARDUnlock.c [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+#define SEC_BA(x) ((u8)(((x) >> 12) & 0x7f))
+
+inline int CARDRand() {
+    next = next * 1103515245 + 12345;
+    return (int)((u32)(next / 65536) % 32768);
+}
+
+/**
+ * @note Address: N/A
+ * @note Size: 0x8
```
Comment (cadmic) [inline src/dolphin/card/CARDUnlock.c]: these `@note`s seem a bit out-of-place

Comment (Yanis42) [reply src/dolphin/card/CARDUnlock.c]: ahh looks like I already removed it

File: src/dolphin/card/CARDStat.c (hunk 1 above)
Comment (Yanis42) [reply src/dolphin/card/CARDStat.c:9]: I hate lowercase hex, I need to figure out something to fix the other instances of it

**Pull request merged**